docs/securing_installation.md

```diff
@@ -0,0 +1,108 @@
+# Securing your Helm Installation
+
+Helm is a powerful and flexible package-management and operations tool for Kubernetes. Installing it using the default installation command -- `helm init` -- quickly and easily installs **Tiller**, the server-side component with which Helm corresponds. 
+
+This default installation applies **_no security configurations_**, however. It's completely appropriate to use this type of installation when you are working against a cluster with no or very few security concerns, such as local development with Minikube or with a cluster that is well-secured in a private network with no data-sharing or no other users or teams. If this is the case, then the default installation is fine, but remember: With great power comes great responsibility. Always use due diligence when deciding to use the default installation.
+
+## Who Needs Security Configurations?
+
+For the following types of clusters we strongly recommend that you apply the proper security configurations to Helm and Tiller to ensure the safety of the cluster, the data in it, and the network to which it is connected.
+
+- Clusters that are exposed to uncontrolled network environments
```

Comment:
Perhaps add "or run untrusted workloads that have access to the network"?

Reply:
clarification: the public network or the internal network?

Reply:
good point. _any_ network that is exposed to workloads, or vice-versa. the point is, we do not know in advance which direction is the concern. If you wanted to be _very_ concerned, both directions are a problem. :-|